frontend/src/components/pages/emotes/create-emote.component.tsx

```diff
@@ -0,0 +1,112 @@
+import { bind } from 'decko';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+// Bootstrap components
+import { Card } from 'react-bootstrap';
+
+import EmoteGuidelines from '../../fragments/emotes/emote-guidelines.component';
+import EmoteDropzone from '../../fragments/emotes/emote-dropzone.component';
```

Comment:
I didn't change any here, but typescript also lets us set a "baseUrl" for the imports.  This means, we can change this from relative to absolute like:

import EmoteGuidelines from 'components/fragments/emotes/emote-guidelines.component';

This makes it a little less crazy with relative paths everywhere.